pkg/apis/events/v1alpha1/storage_types.go

```diff
@@ -107,14 +107,15 @@ const (
 	// TopicReady has status True when the underlying GCP PubSub topic is ready
 	TopicReady apis.ConditionType = "TopicReady"
 
-	// GCSReady has status True when GCS has been configured properly to send Notification events
-	GCSReady apis.ConditionType = "GCSReady"
+	// StorageNotificationReady has status True when GCS has been configured properly to
+	// send Notification events
+	NotificationReady apis.ConditionType = "NotificationReady"
```

Comment:
StorageNotificationReady? That will make it more relevant for end user. II haven't gone over the code in detail but assuming that the notificationJob will also fail in case the bucket doesn't exist.

Reply:
My reasoning is that this is already part of the Storage object, so having StorageNotification is redundant there.